install/migrations/update_10.0.6_to_10.0.7/inventory.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2023 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+/**
+ * @var DB $DB
+ * @var Migration $migration
+ */
+$migration->addField('glpi_printers', 'remote_addr', "string");
+$migration->addField('glpi_networkequipments', 'remote_addr', "string");
+$migration->addField('glpi_unmanageds', 'remote_addr', 'string');
+$migration->addField('glpi_computers', 'remote_addr', "string");
+$migration->addField('glpi_phones', 'remote_addr', "string");
+
+$assets = ['Computer', 'Phone', 'Printer', 'NetworkEquipment'];
+
+foreach ($assets as $itemtype) {
```

Comment:
I guess we should proceed data update only on initial field creation, something like:
```suggestion
$migration->addField('glpi_unmanageds', 'remote_addr', 'string');

$assets = ['Computer', 'Phone', 'Printer', 'NetworkEquipment'];

foreach ($assets as $itemtype) {
    if ($DB->fieldExists($itemtype::getTable(), 'remote_addr')) {
        continue;
    }
    $migration->addField($itemtype::getTable(), 'remote_addr', 'string');
```